docs/i18n.md

```diff
@@ -44,6 +46,12 @@ It compiles the Django messages to the following file:
 
 This is loaded by Django to provide backend translations.
 
+Secondly, it converts any local CSV files (`kolibri_explore_plugin/locale/${locale}/LC_MESSAGES/*.csv`
+to JSON files, allowing for translation to be done in Google Docs via the CSV.
+For this to work, the CSV file must contain `Identifier` and `Translation`
+columns, named exactly that. It may contain other columns (they are ignored),
+and the filename may be anything ending in `.csv`.
```

Comment:
I think we'll have to do some post-processing in the JSON files because of our weird use of translations. See below.